.github/workflows/continuous-integration.yaml

```
@@ -19,12 +19,12 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: [10, 12, 14, 16]
+        node-version: [14, 16, 18]
```

If we make a release on May 2023, we can also drop Node.js v14 as well.

```suggestion
        node-version: [16, 18]
```

see: https://github.com/nodejs/Release/blob/031e2e696365940269a523afe9539ba01e5797a9/README.md#release-schedule